gm4_trapped_signs/pack.mcmeta

```diff
@@ -4,7 +4,6 @@
     "description":"A Gamemode 4 Module"
   },
   "module_name":"Trapped Signs",
-  "module_id":"trapped_signs",
```

Comment:
Why were the "module_id": fields removed? Does the website not read those anymore?

Reply:
I believe that's because, at least for orbis modules, they haven't been updated yet.

Reply:
Removing that field hides the module on the site.